tests/retrieveBearerToken.test.js

```diff
@@ -0,0 +1,136 @@
+import { createMocks } from "node-mocks-http";
+import client from "next-auth/client";
+import { retrieve } from "../pages/api/id/[form]/bearer";
+import queryManager from "../lib/integration/queryManager";
+
+jest.mock("next-auth/client");
+
+describe("Test bearer token retrieve API endpoint", () => {
```

Comment:
Don't see any tests for the case there is an error with the DB connection or something